tests/errors/SettingsError/filters/f01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:--filter;asdf=2016-02-01;
```

Comment:
No it doesn't. It feeds arguments excatly as they are between ';' chars. 
It is testing filter, which has command line is in form:
"--filter" "date=2017-01-01", but in this case, because it's an error test, it is using incorrect filter name. It is excatly same args than what it was *.args file. So it is ok. =)